lmdeploy/pytorch/tools/make_inputs.py

```diff
@@ -157,11 +179,16 @@ def __forward_hook(module, args, kwargs, output):
             target_args = args
             target_kwargs = kwargs
             target_output = output
+            raise ExtractorFound()
```

Comment:
why?

Reply:
This tool is used to extract input/output of a submodule (for debug), computation after the module is not necessary.